int16 bat_cap_
int16 bat_vol_
int16 bat_cur_
int16 bat_error_code_